pastas/stressmodels.py

```diff
@@ -1520,6 +1520,36 @@ def get_water_balance(
         df.index = self.prec.series.index
         return df
 
+    def get_parameters(self, model=None, istress: Optional[int] = None) -> ArrayLike:
```

Comment:
Let's make this method private, ._get_parameters. I don't think users should use this method to get the parameters, because it is very specific for the linear model to get adapted parameters for the response function. So only for internal use.

Reply:
I will also change WellMode.get_parameters to WellMode._get_parameters. Is this ok @dbrakenhoff ?

Reply:
I'm not sure. I use it quite a bit, because quite a few of the Model class' methods do not work out-of-the-box for the WellModel. In order to get the distance added to the 3 Hantush parameters, I use `p = WellModel.get_parameters()`, and pass that onto any model class methods, e.g. `ml.get_step_response("wells", p=p)`.

I think in this case, this method is also useful for other users, and not just for internal use.

Reply:
Then I am in favor of adding a `get_parameters` to all stressmodels (through StressModelBase), which would return the optimal parameters when a model is supplied and the initial values otherwise.